library/ssl_tls.c

```diff
@@ -2226,7 +2220,8 @@ static int ssl_decrypt_buf( mbedtls_ssl_context *ssl )
                                          add_data, 13,
                                          dec_msg, dec_msglen,
                                          dec_msg_result, &olen,
-                                         dec_msg + dec_msglen, taglen ) ) != 0 )
+                                         dec_msg + dec_msglen,
+                                         ssl->transform_in->taglen ) ) != 0 )
```

Comment:
Note: this is inconsistent: we could access the transform as just `transform` here.

Reply:
Note: the inconsistency was fixed in a later commit, please ignore the above comment.